packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -135,12 +164,19 @@ const ToolsPanelHeader = (
 	const defaultItems = Object.entries( menuItems?.default || {} );
 	const optionalItems = Object.entries( menuItems?.optional || {} );
 	const dropDownMenuIcon = areAllOptionalControlsHidden ? plus : moreVertical;
-	const dropDownMenuLabelText = areAllOptionalControlsHidden
-		? _x(
-				'View and add options',
-				'Button label to reveal tool panel options'
-		  )
-		: _x( 'View options', 'Button label to reveal tool panel options' );
+	const dropDownMenuLabelText = sprintf(
+		// translators: %s: The name of the tool e.g. "Color" or "Typography".
+		_x( '%s options', 'Button label to reveal tool panel options' ),
```

Comment:
This works ok, except for the dimensions panel:
![Screen Shot 2022-04-29 at 3 06 33 pm](https://user-images.githubusercontent.com/677833/165899167-e06f0cf6-c9a6-428b-93ad-fae9a9b0431c.png)

Every other tool uses a singular noun, but I understand why 'Dimension' doesn't right.

Reply:
Because it's a mouthful? I think it should be fine: it's the label for a collective group of dimensions, e.g., options of/for dimensions.